projects/packages/search/src/js/dashboard/components/dashboard/index.jsx

```diff
@@ -133,29 +127,27 @@ function SearchDashboard( props ) {
 
 	return (
 		<div className="jp-search-dashboard-page">
-			{ props.isLoading && <LoadingPlaceHolder /> }
-			{ ! props.isLoading && (
+			{ isLoading && (
+				<img
+					className="jp-search-dashboard-page-loading-spinner"
+					width="32"
+					height="32"
+					alt={ __( 'Loading', 'jetpack' ) }
+					src="//en.wordpress.com/i/loading/loading-64.gif"
+				/>
+			) }
```

Comment:
Hi there 👋 
What do you think of using the [RNA spinner](https://github.com/Automattic/jetpack/tree/master/projects/js-packages/components/components/spinner) instead?
It might make sense for gradual introducing of the Emerald design.

Reply:
Here it is in the Storybook: https://automattic.github.io/jetpack-storybook/?path=/docs/playground-spinner--default

Reply:
@sergeymitr oh thanks, I didn't know there's one 😄